### ISSUE TITLE
checkpoint链接失效

### ISSUE THREAD

**Issue by liwead** [Milkomeda98/CKD-TransBTS#11]
github所提到的最好的权重文件checkpoint链接失效，点击以后跳转显示 “抱歉，您无法访问此项目，因为它违反了我们的[服务条款](http://www.google.com/google-d-s/intl/zh-CN/terms.html)。”

**Comment by faizan1234567** [comment]
Yes, @Milkomeda98 @liwead any update on the weight file? Same issue here. I can't access the weight file. I would be nice if you could please provide with the weight file, the link does not work.
